icd/api/compiler_solution_llpc.cpp

```diff
@@ -466,6 +489,19 @@ VkResult CompilerSolutionLlpc::CreateComputePipelineBinary(
     {
         *ppPipelineBinary = pipelineOut.pipelineBin.pCode;
         *pPipelineBinarySize = pipelineOut.pipelineBin.codeSize;
+        if (pipelineOut.pipelineCacheAccess !=
+            Llpc::CacheAccessInfo::CacheNotChecked) {
+          pCreateInfo->pipelineFeedback.feedbackValid = true;
+          pCreateInfo->pipelineFeedback.hitApplicationCache =
+              pipelineOut.pipelineCacheAccess ==
+              Llpc::CacheAccessInfo::CacheHit;
+        }
+        if (pipelineOut.stageCacheAccess !=
+            Llpc::CacheAccessInfo::CacheNotChecked) {
+          pCreateInfo->stageFeedback.feedbackValid = true;
+          pCreateInfo->stageFeedback.hitApplicationCache =
```

Comment:
I think the PAL coding style requires `var = (A == B)` to have parens around the comparison.

Reply:
done